v7/jsonfeed/jsonfeed.py

```diff
@@ -62,17 +62,14 @@ def set_site(self, site):
             'feed_links_append_query': self.site.config['FEED_LINKS_APPEND_QUERY'],
             'feed_length': self.site.config['FEED_LENGTH'],
             'feed_plain': self.site.config['FEED_PLAIN'],
-            'feed_previewimage': self.site.config['FEED_PREVIEWIMAGE'],
+            'feed_previewimage': self.site.config.get('FEED_PREVIEWIMAGE', True),
             'feed_read_more_link': self.site.config['FEED_READ_MORE_LINK'],
             'feed_teasers': self.site.config['FEED_TEASERS'],
             'jsonfeed_append_links': self.site.config.get('JSONFEED_APPEND_LINKS', True),
             'site_url': self.site.config['SITE_URL'],
             'blog_title': self.site.config['BLOG_TITLE'],
             'blog_description': self.site.config['BLOG_DESCRIPTION'],
             'blog_author': self.site.config['BLOG_AUTHOR'],
-            'tag_pages_titles': self.site.config['TAG_PAGES_TITLES'],
```

Comment:
You could make it work with both versions properly by checking if `TAG_PAGES_TITLES` exists and adding the three entries based on that.

Reply:
but AFAICT this key isn't used anyhow.

Reply:
It’s not used directly, but values from the dict will eventually end up here:

https://github.com/getnikola/plugins/blob/765888e315d66180ef79428ab2d01aa15245abae/v7/jsonfeed/jsonfeed.py#L121-L122

`self.kw` is also used to ensure stuff is rebuilt when a config setting is changed.